### ISSUE TITLE
ff_train gives a nonsensical answer if no groundtruth was prepared

### ISSUE THREAD

**Issue by jduijvenbode-WWF** [jduijvenbode-WWF/ForestForesight#39]
**Describe the bug**
If no groundtruth was available for the training data then groundtruth is NA. It then states that the length of label and of the input data is not the same, while it is clear that there was just no groundtruth available. The error should be more concise to help the user

**To Reproduce**
ff_run(train_dates="2024-11-01",country="BRN",save_path="model.model")

**Expected behavior**
it should give an error in the spirit of: no groundtruth labels were available for training. Check if groundtruth data was available for the training dates.
